src/ctap/status_code.rs

```diff
@@ -67,5 +67,11 @@ pub enum Ctap2StatusCode {
     // CTAP2_ERR_VENDOR_FIRST = 0xF0,
     CTAP2_ERR_VENDOR_RESPONSE_TOO_LONG = 0xF0,
     CTAP2_ERR_VENDOR_RESPONSE_CANNOT_WRITE_CBOR = 0xF1,
+
+    /// An internal invariant is broken.
```

Comment:
Three /// ?

Reply:
This means documentation. It's similar to doxygen but with language support, in particular `cargo doc` generates HTML files for the documentation. When writing `///` the documentation is attached to the following item. When using `//!` it is attached to the containing item (usually used for modules).

Reply:
Thanks!